Source/WebKit/Configurations/BaseXPCService.xcconfig

```diff
@@ -80,6 +80,11 @@ WK_PATH_FROM_SERVICE_EXECUTABLE_TO_FRAMEWORK_SHALLOW_BUNDLE_YES_DEPLOYMENT_YES =
 WK_PATH_FROM_SERVICE_EXECUTABLE_TO_FRAMEWORK_SHALLOW_BUNDLE_NO_DEPLOYMENT_NO = ../../..;
 WK_PATH_FROM_SERVICE_EXECUTABLE_TO_FRAMEWORK_SHALLOW_BUNDLE_YES_DEPLOYMENT_NO = ..;
 
+// In relocatable builds, dyld needs to find any Swift compatibility libraries
+// that were copied into the application bundle.
+LD_RUNPATH_SEARCH_PATHS = $(LD_RUNPATH_SEARCH_PATHS_$(WK_RELOCATABLE_FRAMEWORKS));
```

Comment:
I know that we don't use `LD_RUNPATH_SEARCH_PATHS` now, so it doesn't matter if there is an `$(inherited)` or not, but have you considered adding it?